lib/round.rb

```diff
@@ -0,0 +1,42 @@
+class Round
+  attr_reader :deck,
+              :turns
+  def initialize(deck)
+    @deck = deck
+    @turns = []
+  end
+
+  def current_card
+    @deck.cards[turns.count]
+  end
+
+  def take_turn(guess)
+    new_turn = Turn.new(guess, current_card)
+    @turns << new_turn
+    new_turn
+  end
+
+  def number_correct
+    correct_turns.count
+
+  end
+
+  def number_correct_by_category(category)
+    correct_turns.select { |turn| turn.card.category == category }.count
+  end
+
+  def percent_correct
+    ((number_correct.to_f / @turns.count) * 100).truncate()
```

Comment:
I would have much rather seen `round` here in place of `truncate` since with `truncate`, the numbers have the decimal just get chopped off, when `round` would round up or down accordingly.